roosevelt.js

```diff
@@ -220,7 +225,7 @@ module.exports = function (params) {
     }, app.get('params').shutdownTimeout)
 
     app.set('roosevelt:state', 'disconnecting')
-    logger.log('\n💭 ', `${appName} received kill signal, attempting to shut down gracefully.`.magenta)
```

Comment:
I know that `\n` looks silly, but it was there so the log wouldn't look like this:

![silly log](https://user-images.githubusercontent.com/8182831/44220010-6261dd00-a14b-11e8-8e0d-e98829654191.png)